internal/database/actions_test.go

```diff
@@ -56,7 +82,7 @@ func TestMeasurementWorkflow(t *testing.T) {
 	}
 	defer os.RemoveAll(tmpdir)
 
-	sess, err := Connect(tmpfile.Name())
+	database, err := New(tmpfile.Name())
```

Comment:
Sorry for asking to pile up another change but, now that I re-read the API, I think the following:

* `database.New` is idiomatic (it reads well)

* the stdlib uses `sql.Open` (which is the semantics I would expect for files)

* the code previously used `Connect` (which is the name we have in python's sqlite driver)

* the sqlite3 codebase for Go uses `SqliteDriver.Open`

In light of all of this, I would say we should perhaps use `Open` instead of `New`.